### PR TITLE
Export: Export plugin settings

### PR DESCRIPTION
**Pull request by ryantxu**
This adds plugin settings as an exportable option (minus secure JSON!)

Since the number of options keeps growing, this lets the backend define the set of possible flags, and sends them to the frontend

https://user-images.githubusercontent.com/22381771/178796478-f3d51817-1d48-48db-9c1d-42bda4dc38bd.mov

### CODE REVIEW

File: pkg/services/export/types.go [hunk 1]
```diff
@@ -28,7 +28,8 @@ type ExportConfig struct {
 		Services   bool `json:"services"`
 		Usage      bool `json:"usage"`
 		Anno       bool `json:"anno"`
-		Snapshots  bool `json:"snapshots"`
+		//	Snapshots  bool `json:"snapshots"`
```
Comment (nmarrs) [inline pkg/services/export/types.go:31]: Are we temporarily disabling ability to export snapshots or something?

File: public/app/features/storage/ExportView.tsx [hunk 2]
```diff
@@ -78,6 +77,13 @@ export const ExportView = ({ onPathChange }: Props) => {
     getBackendSrv().post('/api/admin/export/stop');
   };
 
+  const setInclude = useCallback(
+    (k: string, v: boolean) => {
+      setBody({ ...rawBody!, include: { ...rawBody!.include, [k]: v } });
```
Comment (nmarrs) [inline public/app/features/storage/ExportView.tsx:82]: This is pretty cool effect (switching toggle updates include json property in real time) - though I guess that is what we are using as a payload to endpoint so makes sense

**Review by nmarrs — APPROVED** [review]
Tested locally and works 💯

File: pkg/services/export/export_plugins.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			Created       time.Time `xorm:"created" json:"created"`
+			Updated       time.Time `xorm:"updated" json:"updated"`
+		}
+
+		rows := make([]*pResult, 0)
+
+		sess.Table("plugin_setting").Where("org_id = ?", helper.orgID)
+
+		err := sess.Find(&rows)
+		if err != nil {
```
[the remark this answers is not in the document]
Comment (ryantxu) [reply pkg/services/export/export_plugins.go:31]: good catch -- added a check for that.  I don't think it is possible though -- this is one of the original (pre-enterprise) SQL tables :)

**Review by nmarrs — COMMENTED** [review]
Refreshed UI looks great! I'll upload a new screencast to PR so that is up to date for documentation purposes but otherwise I think we are good to merge 💯

**Pull request merged**